libkbfs/config_local.go

```diff
@@ -570,6 +570,7 @@ func (c *ConfigLocal) ResetCaches() {
 	c.kcache = NewKeyCacheStandard(5000)
 	// Limit the block cache to 10K entries or 1024 blocks (currently 512MiB)
 	c.bcache = NewBlockCacheStandard(c, 10000, MaxBlockSizeBytesDefault*1024)
+	oldDirtyBcache := c.dirtyBcache
```

Comment:
I didn't do that because it breaks the flow of calculating and then using the vars used in the new dirty bcache.  Is there a reason why moving it is better?